client-generation.ps1

```diff
@@ -0,0 +1,9 @@
+node -v
+if(! $? ){
+	throw "To use Autorest to generate REST client, Node.js is required. Version 10.15.x LTS is preferred. "
+}
+autorest --info  | Out-Null
+if(-not $? ){
+	npm install -g autorest
+}
+autorest --csharp  --v3 swagger/readme.md   
```

Comment:
please add to https://github.com/Azure/azure-signalr/blob/dev/docs/build-source.md later.

Reply:
As the generated file are already in the repo, they don't need to generate themselves.